tests/governance.py

```diff
@@ -91,6 +91,24 @@ def test_no_quote(network, args, notifications_queue=None, verify=True):
     ) as uc:
         r = uc.get("/node/quote")
         assert r.status_code == http.HTTPStatus.NOT_FOUND
+    return network
+
+
+@reqs.description("Check member data")
+def test_member_data(network, args):
+    primary, _ = network.find_nodes()
+    with primary.client("member0") as mc:
+
+        def member_info(mid):
+            return mc.post(
+                "/gov/read", {"table": "ccf.members", "key": mid}
+            ).body.json()
+
+        assert "member_data" not in member_info(0)
+        assert "member_data" not in member_info(1)
+        assert "member_data" not in member_info(2)
+        assert member_info(3)["member_data"] == {"is_operator": True}
```

Comment:
This seems fragile, linked at a distance to the `--initial-operator-count` argument defined in CMake and the default `--initial-member-count`. Can we write this as a loop over `args.initial_member_count` and `args.initial_operator_count`?

Reply:
Yeah, idk, I think that's fragile differently because if the initial_operator_count becomes zero for some reason (for example in an accidental CMake change), then the test passes even though it checks nothing.

I've made the check more generic, but added an assert on the initial count.